sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-energy-demand/automl-forecasting-task-energy-demand-advanced-mlflow.ipynb

```diff
@@ -18,11 +18,11 @@
     "- Connect to your AML workspace from the Python SDK\n",
     "- Create an `AutoML time-series forecasting Job` with the 'forecasting()' factory-fuction.\n",
```

Comment:
Please remove the period at the end of the sentence to be consistent with the rest of the items in this list.

Reply:
Fixed